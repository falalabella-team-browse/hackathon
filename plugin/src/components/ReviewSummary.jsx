import React from 'react';
import styled from 'styled-components';

const Rating = styled.p`
	font-size: 48px;
	font-weight: 900;
`;

const Summary = styled.p`
	font-weight: 700;
	font-size: 16px;
	color: #4f585e;
`;

const ReviewSummary = ({ rating, count }) => {
	return (
		<div>
			<Rating>{rating}</Rating>
			<Summary>based on {count} ratings</Summary>
		</div>
	);
};

export default ReviewSummary;
