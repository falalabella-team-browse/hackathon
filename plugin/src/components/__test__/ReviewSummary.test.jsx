import React from "react";
import { shallow } from "enzyme";
import ReviewSummary from "../ReviewSummary";

describe("ReviewSummary", () => {
  it("should render properly", () => {
    const wrapper = shallow(<ReviewSummary rating={4} count={3} />);
    expect(wrapper).toMatchSnapshot();
  });
});
