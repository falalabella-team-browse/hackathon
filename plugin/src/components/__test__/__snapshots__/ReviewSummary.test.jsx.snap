// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewSummary should render properly 1`] = `ShallowWrapper {}`;
